anaDACScan.py

```diff
@@ -0,0 +1,321 @@
+#!/usr/bin/env python
+
+r"""
+``anaDACScan`` -- analyzes DAC scan data
+========================================
+
+Synopsis
+--------
+
+**anaDACScan.py** [*OPTIONS*]
+
+Description
+-----------
+
+This script reads in calibration information and DAC scan data, performs a fit for each VFAT, computes the DAC value corresponding to the nominal current or voltage for each VFAT, and reports the results.
+
+Arguments
+---------
+
+.. program:: anaDACScan.py
+
+.. option:: infilename
+
+    Name of the input file root.
+
+.. option:: --assignXErrors  
+
+    If this flag is set, the dacValX_Err values stored in the input file are used when constructing the plots and performing the fits. If this flag is not set, the error associated with the dacValX values is always taken to be 0. Note that these errors are actually errors on the variable plotted on the y-axis in the DAC vs ADC plots. The dacValY_Err values, which would correspond to the errors on the x-axis in the DAC vs ADC plots, cannot be used because fits do not converge when they are used. 
```

Comment:
Line 28 is too confusing.  Re-write:

```
If this flag is set then an uncertain on the DAC register value is assumed, otherwise the DAC register value is assumed to be a fixed unchanging value (almost always the case). 
```

Reply:
done